spark-common/src/main/js/src/spark-bootstrap.js

```diff
@@ -20,6 +20,71 @@ function AppLoader() {
         }
     };
 
+    var defaultInlineDialogOptions = {
+        width: '540px',
+        triggerText: 'Inline trigger',
+        alignment: 'bottom left'
+    };
+
+    /**
+     *
+     * @param appName Name of the application to bootstrap
+     * @param appPath application path which will be used to load the necessary angular resources
+     * @param createOptions see defaultInlineOptions (for more details - https://docs.atlassian.com/aui/7.6.3/docs/inline-dialog.html)
+     * @param startedCallback Callback which will be called after the application was successfully started
+     */
+
+    this.loadAppInInlineDialog = function(element, appName, appPath, createOptions, startedCallback) {
+
+        // append trailing slash if not there (before the query string if present)
+        var appPathParts = appPath.split('?');
+
+        var appBasePath = appPathParts[0];
+
+        appBasePath = (/\/$/.test(appBasePath) || /\.html$/.test(appBasePath) ? appBasePath : appBasePath + '/');
+
+        var fullAppPath = AJS.contextPath() + appBasePath + (appPathParts.length > 1 ? '?' + appPathParts[1] : '');
+
+        var sparkAppContainerElementId = appName + '-spark-app-container';
```

Comment:
I think we should have some random component in the ID just to be sure there's no conflict.
This ID seems to be used only in this function so that should be fine.

Reply:
You could also fix it in `openFullscreenIframeDialog` which has some extra `remove` call to work around ID conflicts.
@rikuhaa Any thoughts?